components/coda/sources/row-created/row-created.mjs

```diff
@@ -0,0 +1,105 @@
+import coda from "../../coda.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "coda-row-created",
+  name: "Row Created",
```

Comment:
Source names should start with "New". See https://pipedream.com/docs/components/guidelines/#source-name